crates/turbopack-ecmascript-runtime/js/src/browser/dev/runtime/base/runtime-base.ts

```diff
@@ -386,6 +387,17 @@ function instantiateModule(id: ModuleId, source: SourceInfo): Module {
   return module;
 }
 
+/**
+ * no-op for browser
+ * @param modulePath
+ */
+function resolveAbsolutePath(modulePath?: string): string {
+  if (modulePath) {
+    return modulePath;
+  }
+  return "/ROOT/";
```

Comment:
Shoudn't it return something like:

```suggestion
  return `/ROOT/${modulePath || ''}`;
```

Reply:
Yes, initially thought it was added by codegen